Cargo.toml

```diff
@@ -20,6 +20,7 @@ path = "src/main.rs"
 [dependencies]
 anyhow = "1"
 clap = { version = "3.2", features = ["cargo", "derive", "env"] }
+component = { path = "component" }
```

Comment:
This will need to specify the version as well if we wish to publish future `fuelup` versions to crates.io.